src/main/java/seedu/address/ui/PetCard.java

```diff
@@ -49,9 +49,9 @@ public PetCard(Pet pet, int displayedIndex) {
         this.pet = pet;
         id.setText(displayedIndex + ". ");
         name.setText(pet.getName().fullName);
+        ownerName.setText(pet.getOwnerName().value);
```

Comment:
Maybe reorder the `setText()` method calls to follow the order of the @FXML tags for easier readability?

Reply:
I had initially ordered the `PetCard` method to reflect the tags in order of our mockup UI. 
Nevertheless, I had reordered the **@FXML** tags to match the `PetCard` method for readability.